source/core/src/main/com/csse3200/game/entities/configs/ProjectileConfig.java

```diff
@@ -6,8 +6,6 @@
  * Configuration for projectiles.
  */
 public class ProjectileConfig extends BaseEntityConfig implements Weapon {
```

Comment:
Will this not cause issues? pardon the question, I haven't worked with config files much

Reply:
Hi this will not cause problems due to ProjectileConfig extending BaseEntityConfig (which has health and baseAttack defined)